src/pages/heeyeonKim/Login/Login.js

```diff
@@ -1,7 +1,61 @@
+import React, { useState } from 'react';
 import './Login.scss';
+import { useNavigate } from 'react-router-dom';
 
 const LoginHeeyeon = () => {
-  return <h1>김희연의 로그인 컴포넌트</h1>;
-};
+  const navigate = useNavigate();
+  const [id, setId] = useState('');
+  const saveUserId = e => {
+    setId(e.target.value);
+  };
+
+  const [pw, setPw] = useState('');
```

Comment:
```suggestion
  const navigate = useNavigate();
  const [id, setId] = useState('');
  const [pw, setPw] = useState('');

  const saveUserId = e => {
    setId(e.target.value);
  };

```

state를 한 구역에 위치해 주는 것이 관리 측면이나 가독성 측면에서 더 좋습니다